agent/hook.go

```diff
@@ -37,7 +38,7 @@ func (a *Agent) ExecuteHooks(request *corev2.CheckRequest, status int) []*corev2
 				// code and severity (ex. 0, ok)
 				in := hookInList(hookConfig.Name, executedHooks)
 				if !in {
-					hook := a.executeHook(hookConfig, request.Config.Name)
+					hook := a.executeHook(hookConfig, request.Config.Name, assets)
```

Comment:
Can't different hooks use different assets?

Reply:
Yes they can. Here `assets` is the entire list of assets that are relevant to the request.

Reply:
Similarly to my other comment, can we imagine hooks each with assets that are fine taken as individual sets but somehow incompatible (or worse silently producing a wrong result) when merged into a single set of assets?